content/release-10-17-0/datahub-10-17-0-bundle/10_17_0.md

```diff
@@ -0,0 +1,35 @@
+---
+weight: 40
+title: Release 10.17.0
+layout: bundle
+---
+
+Cumulocity IoT DataHub Release 10.17 includes the following improvements, limitations, and known issues:
+
+### Reporting of failed offloading runs
+
+The offloading configuration provides options for raising alarms in case of a failed offloading or compaction run. When activated and a run fails, a Cumulocity IoT alarm is raised, which will be available in the Cumulocity IoT cockpit and can also be queried via the Cumulocity IoT API.
```

Comment:
```suggestion
The offloading configuration provides options for raising alarms in case of a failed offloading or compaction run. When activated and a run fails, a Cumulocity IoT alarm is raised, which is available in the Cumulocity IoT Cockpit and can also be queried via the Cumulocity IoT API.
```